arrabiata/tests/constraints.rs

```diff
@@ -164,3 +166,46 @@ fn test_gadget_elliptic_curve_scaling() {
 
     helper_check_gadget_activated(instr, Gadget::EllipticCurveScaling);
 }
+
+// This test is mostly an example to show to compute the cross-terms when we do
+// have the expressions, and some evaluations.
+// It doesn't test anything in particular. It is mostly an "integration" test.
```

Comment:
A way to make this test test thing would be to checked the cross term property
poly(a +r*b) = poly(a) + r^n poly(b) + \sum_i r^i cross_term_i(a,b)
Also a nit: I wouldn't call that an integration test

Reply:
"Also a nit: I wouldn't call that an integration test"

That's the reason "integration" is double quoted.